test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -345,6 +343,9 @@ def test_dependency_check_cpu(cpu, ec2_connection):
     reason="Executing test in canaries pipeline during only a limited period of time.",
 )
 def test_dependency_check_gpu(gpu, ec2_connection):
+    # TODO: Fix test on HF inference
+    if "huggingface-tensorflow-inference" in gpu or "huggingface-pytorch-inference" in gpu:
```

Comment:
Are we planning to merge the PR with these tests skipped?

Reply:
Yeah, I've updated the corresponding release GitHub issue to state that we need to run this test manually for release. I've tested manually and the test passes, so we'll need to follow up separately to dive into the infrastructure issue causing this failure